src/content/en/2022/structured-data.md

```diff
@@ -19,37 +19,881 @@ featured_stat_label_3: TODO
 unedited: true
 ---
 
-## TODO
+
+# Structured Data
+
+
+## Web Almanac 2022
+
+
+![alt_text](images/image1.png "image_tooltip")
+
+
+
+# Content team
+
+_**Authors:  Allen ONeill ([allen.oneill@webdataworks.io](mailto:allen.oneill@webdataworks.io)), Andrea Volpini ([andrea@wordlift.io](mailto:andrea@wordlift.io)), John Barrett ([john.barrett@webdataworks.io](mailto:john.barrett@webdataworks.io)), Ananya Pathak ([ananya.pathak@webdataworks.io](mailto:ananya.pathak@webdataworks.io))**_
+
+**_Reviewers: Jono Alderson ([jonoalderson@gmail.com](mailto:jonoalderson@gmail.com)), Rob Teitelman ([seorobt@gmail.com](mailto:seorobt@gmail.com)), Aparna Garimella ([j.aparnaa@gmail.com](mailto:j.aparnaa@gmail.com))_**
+
+_**Analysts:**_
+
+_**Editors: Jasmine Drudge-Willson ([jasmine.a.willson@gmail.com](mailto:jasmine.a.willson@gmail.com))**_
+
+
+# Introduction
```

Comment:
Can you remove everything from line 22 to line 43? Not needed in Markdown.

Also looks like all the headings are one level too high. Introduction should be `## Introduction` since the chapter name (which will automatically be added by the build) is the H1. There’s an option in the markdown converter extension to demote levels by one level to deal with this.